orchestrator/graphql/utils/get_subscription_product_blocks.py

```diff
@@ -88,6 +89,7 @@ def included(key: str, value: Any) -> bool:
                 if included(k, v)
             ],
             in_use_by_relations=product_block.get("in_use_by_relations", []),
+            outside_subscription_boundary=product_block["owner_subscription_id"] == subscription_id,
```

Comment:
Let's wait for @wouter1975 to discuss the exact definition of a subscription boundary

Reply:
Yes, and I would probably add that check to a separate function for clarity and because I'm pretty sure it will get more complicated in the future.

Reply:
This seems to be "going a little too short through the corner" as the Dutch saying goes ;) Lets discuss next week.

Reply:
Seems like this could also be calculated by using the owner_subscription_subscription_id, furthermore we still have a logic error. Outside_subscription_boundary should be `True` when !=

Reply:
Logic should be like:
- if owner(product_block) != subscription_owner and inuse-by(product_block) != subscription_owner: outsideboundary is true

Best illustrated with an example: in case of IP service, which has a PB relation to a Serviceport (SP). The SP has a different subscription-owner thus within subscription boundary.

The SP has PB relation to a node, which has a different subscription_owner. The node is has not PB relation to a PB which has the same subscription_owner, so it falls outside the subscription boundary.